Jenkinsfile

```diff
@@ -15,7 +15,11 @@ pipeline {
         }
         stage('Build') {
             steps {
-                sh 'echo Build'
+                sh 'pip install .'
+                sh 'pip install freezegun'
+                sh 'pip install validators'
+                sh 'pip install ukpostcodeparser'
+                sh 'pip install random2'
```

Comment:
Better add a requirements.txt and use pip to install it from there. This way you can add, remove or update packages easily, modifying a list of packages instead the pipeline.

Reply:
I've planned to fix the dependencies problem. The ticket associated is: https://trello.com/c/o9HnoEl0/157-fix-some-dependencies-in-setuppy . At this moment, installing the dependencies one by one is a temporary fix.